### PR TITLE
feat: 🎸 Hover on bars when color is out of range

### PR DESCRIPTION
**Pull request by dariuszlacheta**

### CODE REVIEW

File: packages/charts/src/charts/bar/utils/chart.utils.ts [hunk 1]
```diff
@@ -328,6 +328,7 @@ export const generateHorizontalStackedBars = ({
         width,
         height: barHeight,
         color: getPaletteColor(keys.indexOf(keyName), colors),
+        colorOutOfRange: !colors[index],
```
Comment (maciejrybaniec) [inline packages/charts/src/charts/bar/utils/chart.utils.ts:331]: Could You please add additional test case that generates a bar with `colorOutOfRange` set to `true`.

File: packages/charts/src/charts/bar/utils/chart.utils.ts [hunk 2]
```diff
@@ -328,6 +328,7 @@ export const generateHorizontalStackedBars = ({
...
         value: data[index][keyName],
       };
```
Comment (maciejrybaniec) [inline packages/charts/src/charts/bar/utils/chart.utils.ts:334]: Additionally could You please modify or create additional story is Storybook that presents this use case.

**Review by maciejrybaniec — CHANGES_REQUESTED** [review]

File: packages/charts/src/charts/bar/utils/chart.utils.ts (hunk 1 above)
Comment (dariuszlacheta) [reply packages/charts/src/charts/bar/utils/chart.utils.ts:331]: Tests added

File: packages/charts/src/charts/bar/utils/chart.utils.ts (hunk 2 above)
Comment (dariuszlacheta) [reply packages/charts/src/charts/bar/utils/chart.utils.ts:334]: New story added

**Review by maciejrybaniec — APPROVED** [review]

**Pull request merged**